### DISCUSSION TITLE
How to Clean Trashbin for All Users via CLI in Nextcloud AIO

### DISCUSSION THREAD

**Discussion by vawaver** [nextcloud/all-in-one#4913] [Questions] [answered]
Hello everyone,

I'm currently running Nextcloud AIO and I'm looking for a way to clean the trashbin for all users via the command line interface (CLI). I've noticed that over time, the trashbin can accumulate a lot of data, and I want to perform a clean-up operation to free up some space.

Could anyone provide a step-by-step guide or the specific command(s) needed to achieve this? Any tips on how to automate this process or best practices to ensure I don't accidentally delete important data would also be greatly appreciated.

**Comment by szaimen** [comment, marked as answer]
Hi, based on https://github.com/nextcloud/all-in-one?tab=readme-ov-file#how-to-run-occ-commands and https://docs.nextcloud.com/server/stable/admin_manual/configuration_server/occ_command.html#trashbin, the following should work:
`sudo docker exec --user www-data -it nextcloud-aio-nextcloud php occ trashbin:cleanup --all-users`

**Comment by vawaver** [comment]
Thank you, it works exactly as I want!
